### ISSUE TITLE
[Feature Request] [SuperUrlBar] Right side neers a bit more padding when bringing the nearest icon over

### ISSUE THREAD

**Issue by antikovt** [JLBlk/Zen-Themes#27]
In Super URL Bar, if "Move 1 button that's directly next to..." is enabled, the icon that's been moved over is just a little too close to the edge. Becomes obvious w/ the History button in particular.

![image](https://github.com/user-attachments/assets/e90efc73-3378-4dae-b93c-654315954da1)

**Comment by JLBlk** [comment]
Yeah, i added the option like last minute and only tested it with 1 extension icon, but you're right, i'll make the padding a bit bigger in the next release.
Thanks for the report!
